### PR TITLE
Bug fix, testing and performance

### PR DESCRIPTION
**Pull request by yacc**
Hey Francesco, please review the changes:
- bumped helmet to 6.0.1 to include a fix on their side (https://github.com/helmetjs/helmet/issues/390)
- fix a bug in how the middleware were called (using helmet(config) instead than middleware now
- refactored tests and added more
- Also added performance test info in the README

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,53 +1,46 @@
...
+const debug = require('debug')('salus');
 const getIPAdress = require("./middlewares/getIPAddress");
 const rateLimiter = require("./middlewares/rateLimiter");
 const csp = require("./middlewares/csp");
 const setNonce = require("./middlewares/setNonce");
 const insertNonce = require("./middlewares/insertNonce");
 
+function skipAssets(config) {
+  let skips = [];
+  if (typeof config.contentSecurityPolicy !== 'undefined') {
```
Comment (sullof) [inline index.js:11]: This seems not precise. What is supposed to be contentSecurityPolicy? An object. Then if would make more sense to check 
```
if (typeof config.contentSecurityPolicy === 'object' && config.contentSecurityPolicy.hasOwnProperty('skipCSP')) {}
```

File: middlewares/insertNonce.js [hunk 2]
```diff
@@ -24,22 +26,29 @@ module.exports = (app, config) => {
   let html;
   let index_file = config.siteIndexFile;
 
+  if (index_file === 'undefined' || typeof index_file !== 'string') {
+    console.log("Salus: skipping nonce bc index file in salus.config.js not set")
```
Comment (sullof) [inline middlewares/insertNonce.js:30]: Not sure about this too

File: middlewares/rateLimiter.js [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
 
 module.exports = (app, config) => {
   if (config["rateLimiter"]) {
+    const windowMs = config["rateLimiter"]["windowsMs"] || 10000;
+    const max = config["rateLimiter"]["max"] || 60
     app.use(
       rateLimiter({
-        windowMs: config["rateLimiter"]["windowsMs"] || 10000,
-        max: config["rateLimiter"]["max"] || 60,
+        windowMs: windowMs,
```
Comment (sullof) [inline middlewares/rateLimiter.js:10]: You can just write `{windowMs}` instead of `{windowMs: windowMs}`

File: middlewares/insertNonce.js (hunk 2 above)
Comment (sullof) [reply middlewares/insertNonce.js:30]: The second is enough, if it is === undefined it is also !== string.

File: index.js (hunk 1 above)
Comment (yacc) [reply index.js:11]: good point, fixed

File: middlewares/insertNonce.js (hunk 2 above)
Comment (yacc) [reply middlewares/insertNonce.js:30]: good point, fixed as well

**Pull request merged**